source/installation_guide/dovecot_pro_releases/dovecot-2.3.10.1-rev2/amzn/2/index.rst

```diff
@@ -52,8 +52,8 @@ OX Dovecot Pro IMAP Proxy Authentication plug-in
 .. literalinclude:: dovecot-2.3.10.1-rev2-imap-proxyauth-plugin.repo
      :language: none
 
-OX Dovecot Pro Pigeonhole Sieve Zimbra compability plug-in
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+OX Dovecot Pro Pigeonhole Sieve Zimbra compatibility plug-in
```

Comment:
I can't tell if this is a real typo from production. The internet only points back to these docs.

If the typo is still present in production, someone should file a bug to the vendor.